test/services/sync/repo/user.repo.integration.test.js

```diff
@@ -289,4 +289,66 @@ describe('user repo', () => {
 			expect(res.classNames).to.eql(classNames);
 		});
 	});
+
+	describe('findImportUsersByName', () => {
+		it('should resolve importUsers with given firstName and lastName', async () => {
+			const testSystem = await testObjects.createTestSystem();
+			const school = await testObjects.createTestSchool();
+			const importUser = await testObjects.createTestImportUser({
+				schoolId: school._id,
+				system: testSystem._id,
+				firstName: 'jon',
+				lastName: 'doe',
+			});
+			const res = await UserRepo.findImportUsersBySchoolAndName(school._id, importUser.firstName, importUser.lastName);
+
+			expect(res.length).to.equal(1);
```

Comment:
when expecting having the importUser from above, the id could be compared instead.